cmake/Modules/FindHaskell.cmake

```diff
@@ -25,97 +26,93 @@ find_program (GHC-PKG_EXECUTABLE ghc-pkg)
 
 set (HASKELL_FOUND 0)
 if (CABAL_EXECUTABLE)
-if (C2HS_EXECUTABLE)
```

Comment:
too deeply nested, better not to reformat it.

Reply:
Why I agree about the nesting, I also think the formatting ”helps” us to realize that we should refactor this code.